sites.json

```diff
@@ -6791,6 +6791,17 @@
         ]
     },
 
+        {
```

Comment:
Please remove some of the whitespace indentation to align with the other braces, like the one above.